Maddux

LOVER6868 opened the pull request.

LOVER6868 committed:
Crouch

LOVER6868 committed:
Crouch

LOVER6868 committed:
Crouch

Finished Character Height Changing when crouching

LOVER6868 committed:
Crouch

Finished Character Height Changing when crouching

iDudiHU committed:
Update On the scene

iDudiHU commented:
There are merge conflict's as a result your branch being behind for example the crouch input was already added on the latest version of the code, I will look into bringing code over and merge it so it does not break anything next week tuesday.

Beqlynd committed:
Spyglass Init commit

First Run of spyglass functionality

DWhittley committed:
Major updates to puzzle scripts

DWhittley committed:
Slight update to add fixed update

Beqlynd committed:
Spyglass Tagged object highlights

So, it's looking like the idea I originally had requires duplicating the tagged object and childing the duplicate under the original, scaling it to be very slightly larger by percentage or pixels and the applying the overlay material with a mostly translucent setting, and then destroying the duplicate when spyglass is no longer active.  What's in here is essentially a copy paste of what I did fo the environment material switch.  It's a lot simpler and functional, if we end up not having time or not caring enough to do the crazy thing.

iDudiHU committed:
Merge pull request #14 from TFS2023Capstone/RebeccaMarett

Spyglass Init commit

iDudiHU committed:
Merge pull request #15 from TFS2023Capstone/David

Major updates to puzzle scripts

DWhittley committed:
puzzle update

DWhittley committed:
Puzzle update

DWhittley committed:
Merge pull request #16 from TFS2023Capstone/David

David

DWhittley committed:
puzzle update

iDudiHU committed:
Merge pull request #17 from TFS2023Capstone/David

puzzle update

LOVER6868 committed:
Basic HUD

Has Basic Popup notification functionality. Attemped to fix some merge errors.

LOVER6868 committed:
Basic HUD

Has Basic Popup notification functionality. Attemped to fix some merge errors.

DWhittley committed:
Puzzle Update

jigsaw working

iDudiHU committed:
Merge pull request #18 from TFS2023Capstone/David

Puzzle Update

LOVER6868 committed:
Merge branch 'Maddux' of https://github.com/TFS2023Capstone/TFS2023CAP into Maddux

# Conflicts:
#	Assets/Scripts/Helpers.meta
#	Assets/Scripts/HiddenWorld/Player/ThirdPersonController.cs
#	Assets/Settings/Input/InputActionAsset.inputactions
#	ProjectSettings/TagManager.asset
